deploy-agent/deployd/common/utils.py

```diff
@@ -45,7 +46,7 @@ def exit_abruptly(status=0):
 
 def touch(fname, times=None):
     try:
-        with file(fname, 'a'):
+        with open(fname, 'a'):
```

Comment:
can we still use "file" in py3? assuming "open" will actually open the file handle and "file" does not.

Reply:
file==open 

https://portingguide.readthedocs.io/en/latest/builtins.html#removed-file

https://docs.python.org/2.7/library/functions.html#file